packages/stylelint-plugin/src/rules/no-invalid-use-of-var-tokens-in-equations.ts

```diff
@@ -70,7 +70,8 @@ const noInvalidEquations = (
           options.reporter({
             message: negatedKaizenVariableMessage,
             node: postcssNode,
-            autofixAvailable: !isVariable(postcssNode),
+            autofixAvailable:
+              !isVariable(postcssNode) && options.language === "scss",
```

Comment:
This was a bug.
We don't support autofixing negated variables in LESS, and even though the code does this correctly (look at line 55), we don't want to report that an autofix exists when in LESS.